spec/factories/user.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:name) { |n| "user#{n}"}
+    email {"#{name}@test.com"}
+    password 'password'
+    end
+end
+
+
+# let(:sender) { User.create(email: 'sender@test.com', password: 'password', name: 'sender') }
+# let(:receiver) { User.create(email: 'receiver@test.com', password: 'password', name: 'receiver') }
```

Comment:
Why are these lines here?

Reply:
Ooops I saved the first way just for reference. Forgot to remove it.